inc/lazyload_replacer.php

```diff
@@ -347,7 +366,9 @@ public function can_lazyload_for( $url, $tag = '' ) {
 		if ( defined( 'OPTML_DISABLE_PNG_LAZYLOAD' ) && OPTML_DISABLE_PNG_LAZYLOAD ) {
 			return $type['ext'] !== 'png';
 		}
-
+		if ( Optml_Tag_Replacer::$lazyload_skipped_images < apply_filters( 'optml_lazyload_images_skip', $this->settings->get( 'skip_lazyload_images' ) ) ) {
```

Comment:
I would use an approach similar to the rest of the filterable settings, one example is get_lazyload_bg_classes to avoid applying the filter each time we call `can_lazyload_for`